jobbergateapi2/tests/apps/users/test_routers.py

```diff
@@ -55,11 +54,12 @@ async def test_search_with_pagination_limit_offset(client):
     assert count[0][0] == 2
 
 
-def test_search_without_results(client):
+@pytest.mark.asyncio
+async def test_list_without_results(client):
     """
-    Test when there is no user matching the search criteria
+    Test when there is no user.
     """
-    response = client.get("/users/?name=Some Name")
+    response = client.get("/users/")
 
     assert response.status_code == status.HTTP_200_OK
```

Comment:
Should we also check that nothing was returned?

Reply:
This is already tested, since len(data) == 0